code/src/App.js

```diff
@@ -1,12 +1,30 @@
-import React from 'react'
-import data from './data.json'
-
-console.log(data)
-
-export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+import React from "react";
+
+import { Artist } from "./Components/Artist/Artist";
+import { Album } from "./Components/Album/Album";
+
+import data from "./data.json";
+
+const renderArtist = artist => {
+  const { id, name, external_urls } = artist;
+  return <Artist key={id} name={name} url={external_urls.spotify} />;
+};

```

Comment:
nice done creating functions. 
there's no need to do it this way, you could do it all in the return. But if you feel this is clearer, it's all good.